### ISSUE TITLE
Out of memory

### ISSUE THREAD

**Issue by tkamth** [PGScatalog/pgsc_calc#63]
Dear team,  

when running I am getting this out of memory error when plink computes the score(see below).  The vcf.gz is relatively small containing 50 samples and ~10M variants.   Any idea how to circumvent this problem (by increasing memory?)

Thanks,

Tony

Command error:
  PLINK v2.00a3.3LM 64-bit Intel (3 Jun 2022)    www.cog-genomics.org/plink/2.0/
  (C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3
  Logging to ALL_additive_0.log.
  Options in effect:
    --memory 8192
    --out ALL_additive_0
    --pfile vzs vcf_ALL
    --score ALL_additive_0.scorefile.gz zs header-read cols=+scoresums,+denom,-fid
    --seed 31
    --threads 2

  Start time: Wed Nov  2 17:18:40 2022
  772698 MiB RAM detected; reserving 8192 MiB for main workspace.
  Using up to 2 compute threads.
  50 samples (0 females, 0 males, 50 ambiguous; 50 founders) loaded from
  vcf_newer_ALL.psam.

  Error: Out of memory.  The --memory flag may be helpful.
  End time: Wed Nov  2 17:18:56 2022

**Comment by nebfield** [comment]
Thanks for reporting the issue! Could you please let me know:

* The full nextflow command you're running
* Which version of `pgsc_calc` you're running
* If your VCF has a lot of annotations

As a temporary fix you could try editing `conf/modules.config`:

```
    withName: PLINK2_VCF {
        ext.args = "--new-id-max-allele-len 100 missing"
       memory = 16.GB 
    }
```

**Comment by smlmbrt** [comment]
@tkamth we just released an updated version of the pipeline (v1.3 : https://github.com/PGScatalog/pgsc_calc/releases/tag/v1.3.0) that may solve your issue? Do let us know how it goes, or if you got it to work we can close the issue.
